packages/vite/src/node/plugins/asset.ts

```diff
@@ -344,18 +344,25 @@ async function fileToBuiltUrl(
   const file = cleanUrl(id)
   const content = await fsp.readFile(file)
 
-  if (shouldInline == null) {
-    shouldInline =
-      !!config.build.lib ||
-      // Don't inline SVG with fragments, as they are meant to be reused
-      (!(file.endsWith('.svg') && id.includes('#')) &&
-        !file.endsWith('.html') &&
-        content.length < Number(config.build.assetsInlineLimit) &&
-        !isGitLfsPlaceholder(content))
-  }
+  const finalShouldInline = (() => {
+    if (config.build.lib) return true
+    if (shouldInline != null) return shouldInline
```

Comment:
Maybe `forceInline` expresses the actual usage?

Reply:
That also works for me 👍